python/tk_multi_publish2/api/item.py

```diff
@@ -662,7 +662,7 @@ def _set_context_r(self, context):
         for child in self.children:
             # If the child's context is not explicitly set,
             # then we need to update it as well
-            if not child._context:
+            if context != child._context:
```

Comment:
After thinking about this some more, I think the original logic is correct. If the child item has a local context override, it should not be updated.

Reply:
Yeah it would have overriden the context for children.